delete-old-branches

```diff
@@ -30,14 +30,14 @@ branch_protected() {
 prefix_filter() {
     local br=${1}
 
-    for prefix in ${INPUT_INCLUDE_PREFIXES//,/ }
+    for prefix in ${INPUT_EXCLUDE_PREFIXES//,/ }
```

Comment:
@hwoarang We could also remove whitespaces from the input. If the `INPUT_EXCLUDE_PREFIXES` is something like this: `prefix1, pref ix2`, then the `for` loop will go through the `prefix1` and `pref`. Maybe we can trim whitespace (or even throw an error for bad input):
```suggestion
    INPUT_EXCLUDE_PREFIXES_NO_WHITESPACE="$(echo -e "${INPUT_EXCLUDE_PREFIXES}" | tr -d '[:space:]')"

    for prefix in ${INPUT_EXCLUDE_PREFIXES_NO_WHITESPACE//,/ }
```

Reply:
> > The action deletes all branches by default except the protected ones.
> > As such, there is no reason to have a specific filter to include more
> > branches into this logic. What we want is a mechanism to _exclude_
> > branches from this logic in order to artifically protect branches which
> > are not marked as protected in the github API.
> 
> @hwoarang It will not fill our need anymore.
> Our need was to protect only a few branches and delete only ones starting by a prefix (`sbe*`, `feature*`), etc. So it was an inclusion mechanism.

@teivah This is contrary to the main idea of this action which is to remove the majority of things with few exclusions. Can you not do the opposite and explicitly add the branches you want to protect?

Reply:
@teorizop we could but sanitizing user input is probably something that needs its own PR